src/main/resources/assets/railcraft/blockstates/tracks/outfitted/kit/throttle.json

```diff
@@ -86,6 +86,17 @@
           "layer1": "railcraft:blocks/tracks/outfitted/kit/throttle_overlay_active"
         }
       }
+      "14": {
+        "textures": {
+          "layer0": "railcraft:blocks/tracks/outfitted/kit/limiter_r",
+        }
+      }
+      "15": {
+        "textures": {
+          "layer0": "railcraft:blocks/tracks/outfitted/kit/limiter_r",
+          "layer1": "railcraft:blocks/tracks/outfitted/kit/throttle_overlay_active"
+        }
+      }
```

Comment:
The six "limiter" texture that never got renamed are intended for reverse mode. If you wanted to add more reverse speeds and do it that way.